content/300-guides/300-upgrade-guides/200-upgrading-to-latest/10-upgrade-to-2270.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: Upgrading to 2.27.0
+tocDepth: 3
+---
+
+<TopBlock>
+
+This page describes manual upgrade steps that you need to perform after [upgrading to Prisma 2.27.0](.) if you want to use the `namedConstraints` Preview feature.
+
+</TopBlock>
+
+## Optional: Enable the <inlinecode>namedConstraints</inlinecode> Preview feature
+
+If you choose to enable the `namedConstraints` Preview feature, choose an workflow (Prisma Migrate or Introspection) and upgrade path that suits your requirements.
+
+### Prisma Migrate workflow
```

Comment:
I fear this does not say much for our users. We use that internally as "a thing", but users might not?

Reply:
I changed it to 'Upgrade paths for Prisma Migrate users' and 'Upgrade path for introspection only users'